test/module/irohad/multi_sig_transactions/completer_test.cpp

```diff
@@ -99,11 +99,11 @@ TEST(CompleterTest, BatchExpirationTestExpired) {
   auto completer = std::make_shared<DefaultCompleter>(std::chrono::minutes(1));
   auto time = iroha::time::now();
   auto tx1 = std::make_shared<MockTransaction>();
-  EXPECT_CALL(*tx1, createdTime()).WillOnce(Return(time));
+  EXPECT_CALL(*tx1, createdTime()).WillRepeatedly(Return(time));
```

Comment:
This and below changes look unclear, could you explain them please?

Reply:
Here we impose no assertions on the number of times the function must be called - it does not matter.